pulpcore/cli/common/generic.py

```diff
@@ -712,6 +712,13 @@ def _type_callback(ctx: click.Context, param: click.Parameter, value: Optional[s
     help=_("Number of repository versions to keep."),
 )
 
+pulp_labels_option = pulp_option(
+    "--labels",
+    "pulp_labels",
+    help=_("JSON dictionary of labels to set on {entity}"),
+    callback=load_json_callback,
+)
```

Comment:
This is the easiest version to allow for this, but maybe we can do better.
Should we for example make a `multiple=True` option that takes `<key> <value>` args and assemble the dict ourselves?
The nice thing about this one is that we can pass the content of a json-file. But we should probably add a little schema validation around it.

Reply:
the multiple=True path is better for the user that's doing a small number of labels as one-offs. If I'm setting up a process where I want more than a couple of labels, or I want a standard set of labels applied to a standard set of things every time, accepting a JSON file is way easier.

I suppose we could have --labels=(JSON or @JSON), **and** a --label=(key value) multiple=True? That covers both usecases. Our code would be responsible for Doing The Right Thing, of course...